doc/containers.md

```diff
@@ -172,7 +172,7 @@ host\_name              | string    | randomly assigned | no        | bridged, p
 hwaddr                  | string    | randomly assigned | no        | all                           | -             | The MAC address of the new interface
 mtu                     | integer   | parent MTU        | no        | all                           | -             | The MTU of the new interface
 parent                  | string    | -                 | yes       | physical, bridged, macvlan    | -             | The name of the host device or bridge
-vlan                    | integer   | -                 | no        | macvlan                       | network\_vlan | The VLAN ID to attach to
+vlan                    | integer   | -                 | no        | macvlan, physical             | network\_vlan | The VLAN ID to attach to
```

Comment:
Should probably also add this to the description: " (if used with a physical nic, can only be used once)"